wpsc-core/js/wp-e-commerce.js

```diff
@@ -1355,22 +1342,17 @@ function submit_change_country(){
 	document.forms.change_country.submit();
 }
 
-// submit the fancy notifications forms.
-function wpsc_fancy_notification(parent_form){
-	if(typeof(WPSC_SHOW_FANCY_NOTIFICATION) == 'undefined'){
-		WPSC_SHOW_FANCY_NOTIFICATION = true;
-	}
-	if((WPSC_SHOW_FANCY_NOTIFICATION === true) && (jQuery('#fancy_notification') !== null)){
-		jQuery('#fancy_notification').css({
-		        position:'fixed',
-		        left: (jQuery(window).width() - jQuery('#fancy_notification').outerWidth())/2,
-		        top: (jQuery(window).height() - jQuery('#fancy_notification').outerHeight())/2
-		    });
-
-		jQuery('#fancy_notification').css("display", 'block');
-		jQuery('#loading_animation').css("display", 'block');
-		jQuery('#fancy_notification_content').css("display", 'none');
-	}
+/**
+ * Submit the fancy notifications forms.
+ *
+ * @deprecated  Use WPEC_Fancy_Notifications.fancy_notification() instead.
+ *
+ * @param  object  parent_form  Form element.
+ */
+function wpsc_fancy_notification( parent_form ) {
+
+	console.log( 'wpsc_fancy_notification() is deprecated. Use WPEC_Fancy_Notifications.fancy_notification() instead.' );
```

Comment:
We should likely only pass this 1) If `window.console && window.console.log` and 2) If WP_DEBUG || WPSC_DEBUG is true.